bond_app/oauth2_state_store.py

```diff
@@ -0,0 +1,63 @@
+import base64
+import json
+import secrets
+from collections import namedtuple
+from google.cloud import ndb
+
+# Information associated with a tokens for refreshing service account credentials.
+OAuth2StateInfo = namedtuple('OAuth2StateInfo', ['nonce'])
```

Comment:
There's finally a better way to do this type of thing:
https://realpython.com/python-data-classes/
ex:
```
from dataclasses import dataclass
@dataclass
class OAuth2StateInfo:
    nonce: str
    
```

Reply:
oh cool! Like Records! Or Case Classes!